torchqdynamics/mesolve.py

```diff
@@ -26,6 +26,8 @@ def mesolve(
     t_save = torch.as_tensor(t_save)
     if exp_ops is None:
         exp_ops = []
+    else:
+        exp_ops = torch.stack(exp_ops)
```

Comment:
Can we keep the type consistent? Maybe something like
```python
if exp_ops is None:
    exp_ops = torch.tensor([])
```

Reply:
Yes good catch!